google/cloud/forseti/scanner/scanners/config_validator_util/cv_data_converter.py

```diff
@@ -30,6 +30,7 @@
 _IAM_POLICY = 'iam_policy'
 _RESOURCE = 'resource'
 
+SUPPORTED_ANCESTORS = ['organization', 'folder', 'project']
```

Comment:
The CAI ancestors list uses plural (organizations, folders, projects), can you have generate_ancestors append the plural version when constructing the list to be more CAI-like?

Reply:
Updated PTAL. https://github.com/forseti-security/forseti-security/pull/3800/files#diff-e90bf6871d0fd206933197d0495ed8a6R75